test/Validate.CFG.cpp

```diff
@@ -91,8 +91,12 @@ class Block {
       : label_(label), body_(), type_(type), successors_() {}
 
   /// Sets the instructions which will appear in the body of the block
-  Block& setBody(std::string body) {
-    body_ = body;
+  Block& setBody(std::string body, bool append = false) {
```

Comment:
I would have made a new method.  appendBody(string).
But meh.  No big deal in test code.